packages/components/stories/Slider.js

```diff
@@ -48,52 +48,56 @@ const actions = [
 		link: true,
 		hideLabel: true,
 	},
+	{
+		id: 'icon4',
+		label: 'Click Me',
+		icon: 'talend-smiley-satisfied',
+		'data-feature': 'action',
+		link: true,
+		hideLabel: true,
+	},
+	{
+		id: 'icon5',
+		label: 'Click Me',
+		icon: 'talend-smiley-satisfied',
+		'data-feature': 'action',
+		link: true,
+		hideLabel: true,
+	},
 ];
 
 const functionToFormat = value => `${value}-test`;
-const functionFormatFloor = value => Math.floor(value);
+const functionFormatFloor = value => `${Math.floor(value)}k`;
```

Comment:
what k ?

Reply:
It was just to look like the visual on frontify. I can remove it.